### ISSUE TITLE
Crash by java.lang.IllegalArgumentException: Invalid key. Must be [a-z0-9/._-]

### ISSUE THREAD

**Issue by SecretlyJealous** [PaperMC/Paper#6821]
### Stack trace

```
Log before crash: https://pastebin.com/JDuUQCjD
Crash report: https://pastebin.com/Ce87Td1h
```

### Plugin and Datapack List

FastAsyncWorldEdit, LuckPerms, PlotSquared, ProtocolLib, spark

### Paper version

This server is running Paper version git-Paper-319 (MC: 1.17.1) (Implementing API version 1.17.1-R0.1-SNAPSHOT) (Git: 975f2e5)

**Comment by lynxplay** [comment]
This looks like you are spawning malformatted items into the world.
"*No response*" is not particularly the best way to reproduce anything, but if something along the lines of spawning malformatted items rings a bell, that is on you.

**Comment by NoahvdAa** [comment]
Related to #6558 and #6559.

**Comment by lynxplay** [comment]
> Related to #6558 and #6559.

Yea presumably just a duplicate anyway. Tho I guess we will wait for a bit more info.

**Comment by SecretlyJealous** [comment]
It's a creative server, I've got some players abusing this exploit to crash the server so I was hoping it could be patched

**Comment by kennytv** [comment]
Duplicate of https://github.com/PaperMC/Paper/issues/6558

**Comment by lynxplay** [comment]
While kenny was a bit faster, yea this is a duplicate then. You will have to wait on the previously linked pr (#6559) or something along the lines to be pulled :+1:
